content/blog/news/_index.fr.md

```diff
@@ -1,5 +1,12 @@
 ---
-title: "News"
-linkTitle: "News"
-weight: 20
----
\ No newline at end of file
+title: "CEO Summit"
+linkTitle: "CEO Summit"
+date: 2023-06-01
+---
+Le 1er juin 2023, se tenait à Potsdam, près de Berlin, le CEO Summit du Memorandum of Understanding entre SNCF et Deutsche Bahn. 
+
+L'open source est l'un des huit sujets identifiés du MoU. Loïc Hamelin, Directeur du Programme, était invité à présenter OSRD devant JP Farandou et R Lutz, PDG des groupes SNCF et DB. 
```

Comment:
On met pas leurs prénoms ici ? Ca fait pas trop familier JP ?

Reply:
I agree, we shouldn't use their fisrt name.